File: fuzz/corpus/fuzz_dart_parser/seed_simple.dart
import 'dart:async';

abstract class Greeter {
  String greet(String name);
}

class Hello implements Greeter {
  final String prefix;

  Hello(this.prefix);

  @override
  String greet(String name) => '$prefix $name';
}

void main() {
  print(Hello('hello').greet('world'));
}

File: fuzz/corpus/fuzz_ruby_parser/seed_simple.rb
require 'set'

module Acme
  class Greeter
    attr_reader :prefix

    def initialize(prefix)
      @prefix = prefix
    end

    def greet(name)
      "#{prefix} #{name}"
    end
  end
end

puts Acme::Greeter.new('hello').greet('world')
